test/test_app.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
Is this file just a copy of test_e2e/test_app.py?

If so, just delete this file.